package.json

```diff
@@ -16,15 +16,15 @@
     "eslint-init": "eslint --init"
   },
   "dependencies": {
-    "agentkeepalive": "^4.2.1"
+    "agentkeepalive": "^4.5.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0 || ^8.2.0",
+    "eslint": "^8.57.0",
```

Comment:
eslint 9?

Reply:
Unfortunately same as the other projects, it'll require node 20, so we "can" have it by using node 20 and keeping this project's package.json engine version set to 16, but this is not a good way to do it...
Or we decide to migrate to node 20 now?